src/cascadia/TerminalSettingsEditor/Profiles_Advanced.xaml

```diff
@@ -94,20 +100,19 @@
                 <local:SettingContainer x:Uid="Profile_CloseOnExit"
                                         ClearSettingValue="{x:Bind Profile.ClearCloseOnExit}"
                                         HasSettingValue="{x:Bind Profile.HasCloseOnExit, Mode=OneWay}"
-                                        SettingOverrideSource="{x:Bind Profile.CloseOnExitOverrideSource, Mode=OneWay}"
-                                        Style="{StaticResource ExpanderSettingContainerStyle}">
-                    <muxc:RadioButtons AutomationProperties.AccessibilityView="Content"
-                                       ItemTemplate="{StaticResource EnumRadioButtonTemplate}"
-                                       ItemsSource="{x:Bind Profile.CloseOnExitModeList, Mode=OneWay}"
-                                       SelectedItem="{x:Bind Profile.CurrentCloseOnExitMode, Mode=TwoWay}" />
+                                        SettingOverrideSource="{x:Bind Profile.CloseOnExitOverrideSource, Mode=OneWay}">
+                    <ComboBox AutomationProperties.AccessibilityView="Content"
+                              ItemTemplate="{StaticResource EnumComboBoxTemplate}"
+                              ItemsSource="{x:Bind Profile.CloseOnExitModeList, Mode=OneWay}"
+                              SelectedItem="{x:Bind Profile.CurrentCloseOnExitMode, Mode=TwoWay}"
+                              Style="{StaticResource ComboBoxSettingStyle}" />
                 </local:SettingContainer>
 
                 <!--  Bell Style  -->
                 <local:SettingContainer x:Uid="Profile_BellStyle"
                                         ClearSettingValue="{x:Bind Profile.ClearBellStyle}"
                                         HasSettingValue="{x:Bind Profile.HasBellStyle, Mode=OneWay}"
-                                        SettingOverrideSource="{x:Bind Profile.BellStyleOverrideSource, Mode=OneWay}"
-                                        Style="{StaticResource ExpanderSettingContainerStyle}">
+                                        SettingOverrideSource="{x:Bind Profile.BellStyleOverrideSource, Mode=OneWay}">

```

Comment:
This doesn't seem right.

Reply:
wait what doesn't seem right?

Reply:
Removing the `Style`. Really even modifying this SettingContainer at all. This doesn't have any RadioButtons in it.

Reply:
🤦